.eslintignore

```diff
@@ -1,2 +1,3 @@
 **/App.test.jsx
-**/package.json
\ No newline at end of file
+**/package.json
+.eslintrc.cjs
```

Comment:
I don't see the reason you have ignored these files. I recommend you to avoid ignoring tests as `eslint` helps to catch bugs early on in development.